packages/pilot/src/components/DetailsHead/style.css

```
@@ -56,3 +56,62 @@
     margin-top: var(--spacing-small);
   }
 }
+
+@media print {
+
+  .content {
+    align-items: baseline;
+    border-bottom: 1px solid var(--color-light-grey-50);
+    justify-content: center;
+    padding: var(--spacing-small) 0 var(--spacing-small) 0;
```

Pode-se abreviar isso para:
```suggestion
    padding: var(--spacing-small) 0;
```